packages/paste-core/components/popover/src/types.ts

```diff
@@ -13,7 +13,7 @@ type PopoverButtonBaseProps = {
 };
 
 export type PopoverButtonProps = PopoverButtonBaseProps &
-  Omit<ButtonProps, "as"> & {
```

Comment:
Adding the Omit is messing with the underlying types. With omit it will not show additional pseudo types for reset in an IDE. Removing it fixes this.

I manually tested setting the "as" prop to be something other than button and it had no effect as button is applied in the button component it renders.